@navikt/core/react/src/form/combobox/SelectedOptions/SelectedOptions.tsx

```diff
@@ -28,7 +28,13 @@ const Option = ({ option }: { option: ComboboxOption }) => {
     );
   }
 
-  return <Chips.Removable onClick={onClick}>{option.label}</Chips.Removable>;
+  return readOnly ? (
+    <Chips.Toggle variant="neutral" checkmark={false} as="div">
+      {option.label}
+    </Chips.Toggle>
+  ) : (
+    <Chips.Removable onClick={onClick}>{option.label}</Chips.Removable>
```

Comment:
The chips are clickable when combobox is disabled

Reply:
I couldn't actually click them in the example story (probably due to `pointer-events: none` earlier in the DOM tree) 🤔... I'll still set them to be the same as when `readOnly`, despite the fact like Sjur said, nobody should really be using disabled in the first place, and _especially_ with predefined values (that you then can't change) 😅 .